llvm/include/llvm/IR/Intrinsics.td

```diff
@@ -2136,7 +2136,14 @@ let IntrProperties = [IntrNoMem, IntrNoSync, IntrWillReturn] in {
                                llvm_metadata_ty,
                                LLVMScalarOrSameVectorWidth<0, llvm_i1_ty>,
                                llvm_i32_ty]>;
-
+  def int_vp_sthreecmp : DefaultAttrsIntrinsic<[ LLVMScalarOrSameVectorWidth<0, llvm_i8_ty> ],
```

Comment:
The `vp` intrinsics are for "vector predication", which is basically about applying an operation only to a subset of vector lanes. These would be "normal" intrinsics. Again, I'd look at something like int_smin or int_sadd_sat.

Reply:
Oh right!

I've added them in that style now.